### PR TITLE
Feedback API

### LINKED ISSUE #45: ContactUs Screen FeedbackForm -> API Repository
### Is your feature request related to a problem? Please describe.

Create API Repository for FeedbackForm on the ContactUs Screen.

### Describe the solution you'd like

1. linking the screen with API calls.

### PR DESCRIPTION
**Pull request by Hard-Coder05**
### Description

**Type of Change:** (Delete irrelevant options)
- Code

**Code/Quality Assurance Only** (Delete irrelevant options)

- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?

tested locally and checked with backend team as well

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged
- [ ] I have added one/more required dependencies
- [x] I have formatted the code using dart format (dartfmt)

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: lib/core/res/strings.dart [hunk 1]
```diff
@@ -6,18 +6,24 @@ class S {
...
+  static const String emailKeySharedPreferences = "email";
   // api keys
   static const String tokenKey = "token";
   static const String passwordKey = "password";
   static const String emailKey = "email";
   static const String firstnameKey = "first_name";
   static const String lastnameKey = "last_name";
   static const String phoneKey = "contact";
+  //postfeedback
+  static const String nameKey = "name";
```
Comment (werainkhatri) [inline lib/core/res/strings.dart]: this is for feedback, the variable should contain this information
```suggestion
  static const String feedbackNameKey = "name";
```

File: lib/screens/home/home_repository.dart [hunk 2]
```diff
@@ -85,6 +86,32 @@ class APIHomeRepository extends HomeRepository {
 
   @override
   Future<void> postFeedback(String feedback) async {
-    return;
+    sharedPreferences.setString(S.nameKeySharedPreferences, "Siddharth");
+    sharedPreferences.setString(S.emailKeySharedPreferences, "smishra1605@gmail.com");
```
Comment (werainkhatri) [inline lib/screens/home/home_repository.dart]: remove

**Review by werainkhatri — CHANGES_REQUESTED** [review]

File: lib/screens/home/home_repository.dart (hunk 2 above)
Comment (Hard-Coder05) [reply lib/screens/home/home_repository.dart]: sorry I used this for checking

File: lib/core/res/strings.dart (hunk 1 above)
Comment (Hard-Coder05) [reply lib/core/res/strings.dart]: yes sir

**Review by werainkhatri — APPROVED** [review]

**Pull request merged**